miden/tests/integration/operations/field_ops.rs

```diff
@@ -651,6 +651,17 @@ proptest! {
 
         build_op_test!(asm_op, &[b as u64]).prop_expect_stack(&[expected as u64])?;
     }
+    #[test]
+    fn exp_proptest(a in any::<u64>(), b in any::<u64>()) {
+        let asm_op = "exp";
+        let base = a;
+        let exp = b;
+        let expected = exp_helper(base, exp);
```

Comment:
We should be able to use a built-in exponentiation function that we have on field elements - e.g., `Felt::new(base).exp(exp)`. Though, I'd probably change the variable name from `exp` to something like `pow`.